src/components/DropZone/DropZone.tsx

```diff
@@ -304,30 +307,58 @@ export class DropZone extends React.Component<CombinedProps, State> {
 
     this.setState({error: this.props.error});
 
-    if (!this.dropNode) {
+    const dropNode: HTMLDivElement | Document | null =
```

Comment:
Typescript should infer the return type so we can omit it

Reply:
Yes! 👍  That's right! Thank you! Will omit that check.